readme/doc_resource.go

```diff
@@ -416,6 +418,23 @@ func (r *docResource) Schema(
 					},
 					"updated_at": schema.StringAttribute{
 						Computed: true,
+						PlanModifiers: []planmodifier.String{
+							otherattributemodifier.StringModifyString(path.Root("body"), "Body", true),
+							otherattributemodifier.StringModifyString(path.Root("body_html"), "BodyHTML", true),
+							otherattributemodifier.StringModifyString(path.Root("category"), "Category", true),
+							otherattributemodifier.StringModifyString(path.Root("category_slug"), "CategorySlug", true),
+							otherattributemodifier.BoolModifyString(path.Root("hidden"), "Hidden", true),
+							otherattributemodifier.Int64ModifyString(path.Root("order"), "Order", true),
+							otherattributemodifier.StringModifyString(path.Root("parent_doc"), "ParentDoc", true),
+							otherattributemodifier.StringModifyString(
+								path.Root("parent_doc_slug"),
+								"ParentDocSlug",
+								true,
+							),
+							otherattributemodifier.StringModifyString(path.Root("slug"), "Slug", true),
+							otherattributemodifier.StringModifyString(path.Root("title"), "Title", true),
+							otherattributemodifier.StringModifyString(path.Root("type"), "Type", true),
+						},
```

Comment:
This section configures the `updated_at` parameter to get updated when any of the listed attributes here change. Due to the typing, this is broken out into separate functions based on the source type and the "other" attribute's type.